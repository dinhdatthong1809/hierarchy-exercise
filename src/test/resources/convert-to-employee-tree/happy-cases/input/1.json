{
    "Pete": "Nick",
    "Barbara": "Nick",
    "Nick": "Sophie",
    "Sophie": "Jonas"
}