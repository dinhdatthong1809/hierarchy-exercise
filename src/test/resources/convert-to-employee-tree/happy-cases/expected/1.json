{
    "Jonas": {
        "Sophie": {
            "Nick": {
                "Pete": {},
                "Barbara": {}
            }
        }
    }
}